fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/impl/transfer/fluid/CauldronStorage.java

```diff
@@ -47,6 +47,10 @@
 public class CauldronStorage extends SnapshotParticipant<BlockState> implements SingleSlotStorage<FluidVariant> {
 	// Record is used for convenient constructor, hashcode and equals implementations.
 	private record WorldLocation(World world, BlockPos pos) {
```

Comment:
(hmm, why are we using this and not vanilla `GlobalPos`?)

Reply:
This was introduced in 1.16 when there was no `GlobalPos`. Don't think it would be shorter with `GlobalPos` anyway as we'd need more code to access the world...